src/main/java/Model/Database/Tables/Table/T_FlatOwner.java

```diff
@@ -0,0 +1,193 @@
+package Model.Database.Tables.Table;
+
+import Model.Database.Support.Assurance;
+import Model.Database.Support.DBTable;
+import Model.Database.Support.DBToHtml;
+import Model.Database.Support.DbConfig;
+import Model.Database.Tables.DbEntity;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.Dictionary;
+
+public class T_FlatOwner extends DbEntity implements DBTable, DBToHtml {
```

Comment:
toto je premenovana tab. User?

Reply:
preco premenovana tab user? nie. to je tak podla navrhu databazy. toto je nedolezity pouzivatel len ako vlastnik bytu ktory nema ziadne projekty